cwms-data-api/src/test/java/cwms/cda/api/DataApiTestIT.java

```diff
@@ -320,6 +328,23 @@ protected static void createTimeseries(String office, String timeseries) throws
         }, "cwms_20");
     }
 
+    protected static void createTimeseries(String office, String timeseries, int offset) throws SQLException {
+        CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+        db.connection((c)-> {
+            try(PreparedStatement stmt = c.prepareStatement(createTimeseriesOffsetQuery);) {
+                stmt.setString(1, office);
+                stmt.setString(2, timeseries);
+                stmt.setInt(3, offset);
+                stmt.execute();
+            } catch (SQLException ex) {
+                if (ex.getErrorCode() == 20003) {
+                    return; // TS already exists. that's find for these tests.
```

Comment:
you copy pasted my typo. `find for` -> `fine for`

Reply:
I think this was Ryan's PR....

Reply:
Entirely Possible. It was just in this PR that I caught it.